docs/modules/airflow/pages/usage.adoc

```diff
@@ -1,150 +0,0 @@
-= Usage
```

Comment:
This was a duplicate page left over, I guess?

Reply:
Yes